### PR TITLE
feat: show smart wearable badge + filter by smart wearables

### PR DESCRIPTION
**Pull request by cazala**
This PR:

+ Adds a Smart Wearable badge to `AssetCard`
+ Adds a Smart Wearable badge to `ItemDetailPage` and `NFTDetailPage`
+ Adds a Smart Wearable filter to the `AssetBrowse` component, to filter for only wearables that are smart.
+ The filter is also supported in mobile, where it's shown as a `Radio` to keep it consistent with the "On Sale" one.

Screen Shot 2022-01-18 at 13 54 32

Screen Shot 2022-01-18 at 13 54 42

Screen Shot 2022-01-18 at 13 54 58

### CODE REVIEW

**Review by fzavalia — CHANGES_REQUESTED** [review]
image

With the new tag on mobile it seems to overflow horizontally. Would be nice to wrap underneath.

**Comment by fzavalia** [comment]
Just by adding `flex-wrap: wrap` to `.BaseDetail .left .badges` will make it look like this

image

File: webapp/src/components/Vendor/NFTFilters/FiltersMenu/FiltersMenu.tsx [hunk 1]
```diff
@@ -88,6 +105,39 @@ const FiltersMenu = (props: Props) => {
...
         />
+        <Mobile>
+          <Header sub>{t('nft_filters.smart_wearables')}</Header>
+          <Radio
+            className="smart-toggle-mobile"
+            toggle
+            checked={isOnlySmart}
+            onChange={handleOnlySmartClick}
+          />
+          {/* <div
```
Comment (fzavalia) [inline webapp/src/components/Vendor/NFTFilters/FiltersMenu/FiltersMenu.tsx:116]: Clean comment

Comment (fzavalia) [inline webapp/src/components/Vendor/NFTFilters/FiltersMenu/FiltersMenu.tsx:116]: Delete comment

**Review by fzavalia — CHANGES_REQUESTED** [review]

File: webapp/src/components/Vendor/NFTFilters/FiltersMenu/FiltersMenu.tsx (hunk 1 above)
Comment (cazala) [reply webapp/src/components/Vendor/NFTFilters/FiltersMenu/FiltersMenu.tsx]: ✅

**Comment by cazala** [comment]
Ni catch @fzavalia! I just pushed the fixed styles ✅

**Review by fzavalia — APPROVED** [review]
LGTM

**Pull request merged**